### ISSUE TITLE
"Schedule" in NOTAMs is ignored.

### ISSUE THREAD

**Issue by kebekus** [Akaflieg-Freiburg/enroute#364]
At the moment, Enroute Flight Navigation ignores the field "D) Schedule (optional)" of a NOTAM. The following NOTAM is therefore not correctly interpreted:
```
E3674/23 NOTAM  

Q) EDGG/FALC/IV/NBO / A  /000/999/5020N00732E 
A) EDRK 
B) 2023-12-24 00:00 C) 2024-01-01 23:59  
D) DEC 24 25 26 31 JAN 01 H24
E) AD CLSD.
```
Enroute presents the NOTAM as if the aerodrome was closed 24Dec23-01Jan24, which is not the case.

**Comment by kebekus** [comment]
@schneisi Brace for impact! The problem is easy to solve. Be prepared for a new release in the coming days.

**Comment by kebekus** [comment]
Fixed in 2.29.52.

@schneisi I created a new release in the main branch and uploaded binaries to Google Play and FlatHub. Since the matter with NOTAMs is somewhat important, could you create new binaries and upload them the to app store? -- Thanks!
